lib/avo/base_action.rb

```diff
@@ -145,8 +145,8 @@ def succeed(text)
       self
     end
 
-    def fail(text)
-      add_message text, :error
+    def fail(text, keep_modal_open: false)
```

Comment:
@adrianthedev I don't like this name, what do you think about `persistent_error`?

Reply:
i mean, a different method called `def persistent_error`

Reply:
and maybe we should duplicate that fail method to `def error` and start printing deprecation messages inside `def fail` method because it's a ruby reserved method.